src/components/Header/Header.test.tsx

```diff
@@ -0,0 +1,18 @@
+import { beforeEach, test, expect } from "vitest";
+import { render, screen } from "@testing-library/react";
+import Header from "./Header";
+
+beforeEach(() => {
+  render(<Header />);
+});
+
+test("renders the header element", () => {
```

Comment:
Unnecessary suffix. Everything is an element in HTML.

```suggestion
test("renders the page header", () => {
```